server/model/maintenance_timeslot.js

```diff
@@ -0,0 +1,46 @@
+const { BeanModel } = require("redbean-node/dist/bean-model");
+const { R } = require("redbean-node");
+const dayjs = require("dayjs");
+
+class MaintenanceTimeslot extends BeanModel {
+
+    async toPublicJSON() {
+
+    }
+
+    async toJSON() {
+
+    }
```

Comment:
Is this correct? If so, might it be worth just adding a note explaining why, just looks a little odd when looking through.

Reply:
It is not ready yet. I am trying to add new database tables for recurring.
I change it to draft first.